npm-packages/docs/docs/quickstart/nextjs.mdx

```diff
@@ -172,7 +170,7 @@ version of this quickstart.
   </Step>
 
   <Step title="Start the app">
-    Start the app, open [http://localhost:3000](http://localhost:3000) in a browser,
+    Run your development server, open [http://localhost:3000](http://localhost:3000) in a browser,
```

Comment:
since "development server" is an overloaded term in convex I'll add "Next.js" or "frontend" here